src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandStatus.java

```diff
@@ -10,6 +10,9 @@ public enum CommandStatus {
   /** The element has the count of deleted documents */
   @JsonProperty("deletedCount")
   DELETED_COUNT,
+  /** Status for reporting existing namespaces. */
+  @JsonProperty("existingNamespaces")
```

Comment:
(nit) Same as with the other review, maybe just `"namespaces"`

Reply:
Done.